_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -102689,6 +102736,13 @@
 /obj/structure/cable,
 /turf/open/floor/iron,
 /area/commons/storage/tools)
+"vxe" = (
+/obj/structure/window/reinforced{
+	dir = 1;
+	layer = 2.9
```

Comment:
is this layer intentional?

Reply:
Not sure why that's there but I can remove the layer.